data/com.github.tchx84.Flatseal.appdata.xml.in

```diff
@@ -32,10 +33,12 @@
   <update_contact>martin.abente.lahaye@gmail.com</update_contact>
   <!-- TRANSLATORS: Don't translate this text -->
   <developer_name>Martin Abente Lahaye</developer_name>
+  <developer id="com.github.tchx84">
```

Comment:
I assume this should be consistent between all of my applications right? If so, then probably I should stick to something like `dev.tchx84` ? Unless there _is_ a strong relationship between app-id and this developer-id

Reply:
Yes, it has to be consistent between your applications and eventually a reverse DNS id of a domain you own. There is no relation between app-id and developer id.